Week1/homework/js-exercises/loghello.js

```diff
@@ -0,0 +1,15 @@
+
+let i = 0;
+var hello = [
+        Halo, dunia! // Indonesian,
```

Comment:
What is this? Have you even tested this? This code does not work because there are syntax error. Always test your code before commiting it